core/shared/src/main/scala/me/shadaj/scalapy/interpreter/CPythonInterpreter.scala

```diff
@@ -309,6 +311,39 @@ object CPythonInterpreter {
     ), java.nio.charset.Charset.forName("UTF-8"))
   }
 
+  private object Traceback {
+    private var moduleOpt = Option.empty[py.Module]
+    private def module = moduleOpt
+      .filter(_.toString != "<NULL>")
```

Comment:
Added this, as the cached `py.Module` instance seem to be unloaded from time to time (with its `toString` method returning `<NULL>` when that's the case). Not sure what causes this…